lib/t/matches.ex

```diff
@@ -894,4 +895,65 @@ defmodule T.Matches do
     )
     |> where([m, e, c], is_nil(c.timestamp))
   end
+
+  def check_matches_calls() do
+    matches_without_successful_calls =
+      Match
+      |> join(:inner, [m], c in Call,
+        on:
+          (c.caller_id == m.user_id_1 and c.called_id == m.user_id_2) or
+            (c.caller_id == m.user_id_2 and c.called_id == m.user_id_1)
+      )
+      |> where([m, c], c.accepted_at >= fragment("now() - interval '80 hours'"))
+      |> select([m], m.id)
+
+    matches_for_push =
```

Comment:
same here, not all the matches inside this query will receive a push (e.g. matches with successful calls will be included here as well, but we won't send them any pushes)